apps/addons/models.py

```diff
@@ -2341,4 +2341,14 @@ def track_status_change(old_attr={}, new_attr={}, **kw):
     new_status = new_attr.get('status')
```

Comment:
Doesn't this function get an `instance` in `**kw`? I think that would be better than passing a `dict` to `track_addon_status_change` here and using `instance.__dict__` in the other call site.